packages/parser/src/transformations/categorical/charts/chart-transformation.ts

```diff
@@ -1,17 +1,17 @@
 import { fillWithEmptyKeys } from '../../../utils';
 
 import { GroupByResult, ParserSettings } from '../../../types';
-import { KEEN_KEY } from '../../../constants';
+import { KEEN_KEY, PROPERTIES_CONNECTOR } from '../../../constants';
 
 /**
- * Bar transformation
+ * Chart transformation
  *
  * @param result - categorical results
  * @param parserSettings - parser settings
  * @return transformed results
  *
  */
-export const barTransformation = (
+export const chartTransformation = (
```

Comment:
I suggest to re-name it to something like: `categoricalChartTransformation` as in next iterations we are going to add another transformations related with specific charts.